docs/guides/private-submodules.rst

```diff
@@ -1,92 +1,106 @@
-  - Delete the key added by ``Read the Docs Commercial (readthedocs.com)``
+#. Create a GitHub user and give it read only permissions to all the necessary repositories.
+   You can do this by adding the account as:
 
-- Create a GitHub user and give it read only permissions to all the necessary repositories.
-  You can do this by adding the account
-  as a `collaborator <https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/inviting-collaborators-to-a-personal-repository>`__,
-  as an `outside collaborator <https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/adding-outside-collaborators-to-repositories-in-your-organization>`__,
-  or to a `team in an organization <https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/adding-organization-members-to-a-team>`__.
-- Attach the public SSH key from your project on Read the Docs to the GitHub user you just created
+   - A `collaborator <https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/inviting-collaborators-to-a-personal-repository>`__
+   - An `outside collaborator <https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/adding-outside-collaborators-to-repositories-in-your-organization>`__
+   - Or a `team in an organization <https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/adding-organization-members-to-a-team>`__
 
-  - Go to the user's settings
-  - Click on :guilabel:`SSH and GPG keys`
-  - Click on :guilabel:`New SSH key`
-  - Put a descriptive title and paste the :ref:`public SSH key from your Read the Docs project <guides/private-submodules:Project's SSH key>`
-  - Click on :guilabel:`Add SSH key`
+#. Attach the public SSH key from your project on Read the Docs to the GitHub user you just created
 
-Refer to the `GitHub documentation <https://developer.github.com/v3/guides/managing-deploy-keys/#machine-users>`__ for more information about machine users.
+   - Go to the user's settings
+   - Click on :guilabel:`SSH and GPG keys`
+   - Click on :guilabel:`New SSH key`
+   - Put a descriptive title and paste the
+     :ref:`public SSH key from your Read the Docs project <guides/private-submodules:Copy your project's SSH key>`
+   - Click on :guilabel:`Add SSH key`
 
 GitLab
-------
+~~~~~~
 
-For GitLab, Read the Docs uses `deploy keys with read only access <https://docs.gitlab.com/ee/ssh/#deploy-keys>`__.
-GitLab allows you to reuse a deploy key across different repositories.
+For GitLab, Read the Docs uses `deploy keys with read only access <https://docs.gitlab.com/ee/ssh/#deploy-keys>`__,
+which allows you to reuse a SSH key across different repositories.
 Since Read the Docs already added the public SSH key on your main repository,
 you only need to add it to each repository of your submodules.
 
-- Go to the project's settings on GitLab
+- Go to the project of your submodule on GitLab
```

Comment:
I think it's better to use numbered items here (similar to GitHub) because they are steps in order. Besides, if you are following this guide and changing tabs, it's easier to visually jump to `3. ` than to read the text of the bullet to realize if you already did that step or not.